web-next/app/zhipu/zai-glm-best-practices/page.module.css

```
@@ -0,0 +1,373 @@
+  --color-text-info: #7aa2f7;
+  --color-text-danger: #f7768e;
+  --color-text-success: #9ece6a;
+  --color-text-warning: #e0af68;
+  --color-border-primary: #2b2b34;
+  --color-border-secondary: #232329;
+  --color-border-tertiary: #1c1c23;
+  --font-sans:
+    var(--font-inter), -apple-system, BlinkMacSystemFont, "Segoe UI", "Hiragino Kaku Gothic ProN",
+    "Noto Sans JP", sans-serif;
+  --font-mono:
+    var(--font-mono), ui-monospace, SFMono-Regular, "SF Mono", Menlo, Consolas, monospace;
```

Stylelint (17.14.1)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)